### PR TITLE
SW-13107 - Added CSRF documentation

### PR DESCRIPTION
**Pull request by janbuecker**

### CODE REVIEW

**Comment by tiagojsag** [comment]
@janbuecker IMO there's something missing here. You give a formal description of what an CSRF attack is, and then cover the solution, but there is no logical connection between the two. If I am new to the topic, it will be confusing for me what exactly happens in a real world CSRF attack, and how the token can prevent it. So please:
- After the formal description, please add a small, real world example of how an actual CSRF attack works.
- Once the attack vector is described, you should explain what the solution is: tokens

Once you add these two, your description of the CSRF token provider included in Shopware should come a lot more naturally to the reader

**Comment by tiagojsag** [comment]
@janbuecker what's the status here?

**Comment by janbuecker** [comment]
Will be continued after some other important tasks.

**Comment by janbuecker** [comment]
@tiagojsag you can start reviewing it again please.

**Comment by janbuecker** [comment]
@dnoegel Tiago is on holidays, maybe you can take a look at it?

**Comment by tiagojsag** [comment]
I will be back on the 28th. Also, unless someone gets a Shopware shop for Christmas, I doubt that a lot of people will be reading this. Can't it wait?

File: source/developers-guide/index.html [hunk 1]
```diff
@@ -45,5 +46,5 @@
     <li><a href="{{ site.url }}/developers-guide/backend-components/associations/">Backend Components - Associations</a></li>
     <li><a href="{{ site.url }}/developers-guide/backend-components/listing-extensions/">Backend Components - Listing extensions</a></li>
     <li><a href="{{ site.url }}/developers-guide/backend-components/batch-processes/">Backend Components - Batch processes</a></li>
-    <li><a href="{{ site.url }}/developers-guide/backend-extension/">Extending the backend</a></li>
```
Comment (tiagojsag) [inline source/developers-guide/index.html]: ??

Comment (janbuecker) [inline source/developers-guide/index.html]: Seems to be the result of a merge conflict. Added it back again.

**Comment by tiagojsag** [comment]
https://github.com/shopware/devdocs/pull/210

**Pull request closed without merging**